lib/auth_app/accounts.ex

```diff
@@ -261,93 +342,19 @@ defmodule AuthApp.Accounts do
     if user.confirmed_at do
       {:error, :already_confirmed}
     else
-      {encoded_token, user_token} = UserToken.build_email_token(user, "confirm")
+      {encoded_token, user_token} = UserToken.build_email_token(user, "login")
       Repo.insert!(user_token)
       UserNotifier.deliver_confirmation_instructions(user, confirmation_url_fun.(encoded_token))
     end
   end
```

Comment:
I think this function can be gone too?

Reply:
We want to have different emails for confirmation (first login) vs. regular login, no?

Reply:
Maybe we deal with this inside the UserNotifier by checking for `confirmed_at`, reducing the API surface?